fa25-team-b/cleaners/applicant_property_geocoding.py

```
@@ -0,0 +1,132 @@
+
+    props = (
+        df["Application Property"]
+        .dropna()
+        .astype(str)
+        .drop_duplicates()
+    )
+
+    geolocator = Nominatim(user_agent="property_geo", timeout=20)
+    rows = []
+
+    for i, prop in enumerate(props, start=1):
```

Ruff (0.14.6)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)